README.md

```diff
@@ -32,9 +32,9 @@ Depending on your distribution, you will need to run one of the following comman
 
 ## Note for Windows users running Linux on WSL
 
-ZCLI's credential manager has a dependency on Linux's windowing manager (X11) which will make some commands fail to run on WSL, unless you run a setup of WSL with support for X11/GUI apps. These commands include: `zcli login`, `zcli logout` and `zcli profiles`.
+ZCLI's credential manager has a dependency on Linux's windowing manager (X11) which will cause some commands fail to run on WSL, unless you run a setup of WSL with support for X11/GUI apps. These commands include: `zcli login`, `zcli logout` and `zcli profiles`.
```

Comment:
nit: `which will cause some commands fail` -> `which will cause some commands to fail`